compiler/ballerina-lang/src/main/java/io/ballerina/projects/ModuleContext.java

```diff
@@ -59,7 +59,7 @@
  *
  * @since 2.0.0
  */
-class ModuleContext {
+public class ModuleContext implements io.ballerina.projects.internal.ModuleContext {
```

Comment:
We shouldn't make this public. The idea is to write a public wrapper at `io.ballerina.projects.internal.ModuleContext` to be able to access the necessary getters in this package-private class. Making this public will expose it to the user since we export the `io.ballerina.projects package`.

Reply:
This is a mistake, I will resolve this.